### PR TITLE
[TG-144][feat] Add dashboard instance and REST for working with it

### PR DESCRIPTION
**Pull request by artem713**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/26" target="_blank">ReviewNinja

File: src/API/dashboard/dashboard.routes.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const router = express.Router();    // eslint-disable-line new-cap
+
+router.route('/')
+    /** GET /api/dashboard - Get dashboard */
+    .get(dashboardCtrl.query)
+
+    /** POST /api/dashboard - Create new dashboard */
+    .post(dashboardCtrl.create);
+
+router.route('/:id')
```
Comment (kosich) [inline src/API/dashboard/dashboard.routes.js]: !resolve
that I'm not quite getting: do we have ids for dashboard? can we add/delete dashboards?
dashboard itself for in my opinion shouldn't be RESTful (unless its settings for it, and still its debatable)
please, explain the flow with dashboard

Comment (artem713) [inline src/API/dashboard/dashboard.routes.js]: !resolved in the next commit e7190acb0516f9bf7e93c0f4346b6ce9b21eadb4

**Comment by kosich** [comment]
⭐

**Pull request merged**